resources/setups/common/base_config.json

```diff
@@ -49,7 +49,18 @@
       "enabled": true
     },
     "dot1x": {
-      "enabled": true
+      "enabled": true,
+      "interface": "eth0",
+      "radius": {
+          "secret": "SECRET",
+          "id": "02:42:ac:18:00:70",
+          "radius_socket_info": {
+              "listen_ip": "10.20.0.3",
```

Comment:
This doesn't seem like something that can be hard-coded. Semantically, what is it supposed to be?  You will likely have to find some way to dynamically determine what it should be.

Reply:
Added a way to read it by default.